root/etc/s6-overlay/s6-rc.d/init-mod-transmission-transmission-web-control-install/run

```diff
@@ -3,4 +3,8 @@
 
 printf "/themes/transmission-web-control" > /var/run/s6/container_environment/TRANSMISSION_WEB_HOME
 
+mv /usr/share/transmission/public_html/index.html /usr/share/transmission/public_html/index.original.html
+cp -R /usr/share/transmission/public_html/* /themes/transmission-web-control/
+mv /usr/share/transmission/public_html/index.original.html /usr/share/transmission/public_html/index.html
```

Comment:
To minimise the risk of breakage now and in future it might be preferable to do this as:

```shell
cp /usr/share/transmission/public_html/index.html /themes/transmission-web-control/index.original.html
cp -nR /usr/share/transmission/public_html/* /themes/transmission-web-control/
```